### PR TITLE
Create issue_template.md

### LINKED ISSUE #1407: Issue template
We should do this to help people to create issues https://help.github.com/articles/creating-an-issue-template-for-your-repository/

### PR DESCRIPTION
**Pull request by valib**
This will close #1407

### CODE REVIEW

**Comment by SubJunk** [comment]
Looks good to me, feel free to merge

**Comment by SubJunk** [comment]
( This implements #1407 )

**Comment by Nadahar** [comment]
Are you guys really going to commit those few lines as 6 commits?

**Comment by valib** [comment]
@Nadahar of course not. I will rebase it into one.

**Comment by Nadahar** [comment]
I've made some suggestions too, mostly formatting.

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+In order to help diagnose and fix your issue we often need your "debug information". It's easy to save this information and attach it to a GitHub issue:
+
+  * Start UMS.
+  * Go to the `Logs` tab.
+  * Press the `Create TRACE logs` button which restarts UMS in the "TRACE" mode. If that doesn't work for some reason, either select `Trace` from the `Log Level` dropdown at the bottom or set `log_level = TRACE` in the configuration file and restart UMS manually.
+  * Reproduce the problem.
+  * Click `Pack debug files` on the lower left.
+  * Click `Zip selected files`.
+  * Save the zip file to a location you will remember.
+  * Attach the zip file to the GitHub issue.
```
Comment (Sami32) [inline issue_template.md]: Sadly nothing is said about the size limit...
It should also have make some sens to first update _CONTRIBUTING.md_, imho...

Comment (valib) [reply issue_template.md]: @Sami32 good point. I see that we made something what is already there.

**Comment by SubJunk** [comment]
@Nadahar that's what the "Squash and merge" button is for

@Sami32 @valib I was thinking instead of the CONTRIBUTING.md since this is harder to ignore than that one, what do you think?

**Comment by Nadahar** [comment]
@SubJunk I know what the button does, and I also know that it often isn't used by whoever pushes it. All I wanted to assure was that it was either rebased as @valib has already done, or that it was merged using squash.

**Comment by Sami32** [comment]
@SubJunk Thank you for sharing your thoughts ;)
At least it shouldn't be worst. From my forum experience, i don't have illusion about it, but it will have a better visibility than _CONTRIBUTING.md_, so if some additional size limit and supported formats is added, that is fine for me.

**Comment by Sami32** [comment]
This will closes #1407

**Comment by valib** [comment]
@SubJunk not related to this PR but for me is missing the PR #1416. GitHub responds "Page not found".

**Pull request merged**